### DISCUSSION TITLE
Release binary for ARM (Apple Silicon)

### DISCUSSION THREAD

**Discussion by felixgeissler** [minio/minio#12908] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
On OSX running on apple silicon, I am not able to install minio.

**Describe the solution you'd like**
It would be great if we could get a ARM/Apple silicon build

**Describe alternatives you've considered**
-

**Additional context**
`$brew install minio/stable/minio`
```
...
==> Installing minio from minio/stable
minio: The x86_64 architecture is required for this software.
Error: An unsatisfied requirement failed this build.

```

**Comment by harshavardhana** [comment, marked as answer]
> On OSX running on apple silicon, I am not able to install minio.

You are not able install using "homebrew" - you can although install using https://dl.minio.io/server/minio/release/darwin-arm64/minio

**Comment by felixgeissler** [comment]
> > On OSX running on apple silicon, I am not able to install minio.
> 
> You are not able install using "homebrew" - you can although install using https://dl.minio.io/server/minio/release/darwin-arm64/minio

Works like a charm! Thanks.
Would be nice if we can have this link on the download page:
https://min.io/download#/macos

**Comment by harshavardhana** [comment]
We will support homebrew based installation on arm64 in future, for now use the direct link to the binaries.

**Comment by johndbritton** [comment]
You should be able to install on Apple Silicon using Homebrew now that  https://github.com/minio/homebrew-stable/pull/38 is merged.
